packages/registration_delivery/lib/blocs/search_households/individual_global_search.dart

```diff
@@ -215,6 +223,10 @@ class IndividualGlobalSearchBloc extends SearchHouseholdsBloc {
           projectBeneficiariesList, taskList, sideEffectsList, referralsList);
 
       taskList = tasksRelated[0];
+      taskList.sort((a, b) =>
+          a.clientAuditDetails?.createdTime
```

Comment:
Please change to modified Time instead of createdTime

Reply:
Done